torch/_C/__init__.pyi.in

```diff
@@ -1515,8 +1515,8 @@ def _get_function_stack_at(idx: _int) -> Any: ...
 def _len_torch_function_stack() -> _int: ...
 def _set_torch_dispatch_mode(cls: Any) -> None: ...
 def _push_on_torch_dispatch_stack(cls: TorchDispatchMode) -> None: ...
-def _pop_torch_dispatch_stack(mode_key: _TorchDispatchModeKey | None = None) -> Any: ...
-def _get_dispatch_mode(mode_key: _TorchDispatchModeKey | None) -> Any: ...
+def _pop_torch_dispatch_stack(mode_key: _TorchDispatchModeKey | None = None) -> TorchDispatchMode | None: ...
```

Comment:
Is it possible for this function to return `None`? Seems like [neither branch](https://github.com/pytorch/pytorch/blob/main/torch/csrc/autograd/init.cpp#L1425-L1427) is None-return (`TorchDispatchModeTLS::pop_stack()` raises exception instead of return `None` on empty stack).